src/main/java/teammates/ui/template/FeedbackResponseRow.java

```diff
@@ -67,8 +69,23 @@ public FeedbackResponseRow(int fbIndex, int personIndex, String personType,
                     }
                 }
                 responseRow.setVisibilityIcon(isVisibilityIconShown, whoCanSeeComment);
-                responseRow.enableEditDelete();
-                this.responseComments.add(responseRow);
+
+                boolean isStudentGiver = results.roster.isStudentInCourse(giverEmail);
+                if (isStudentGiver) {
+                    this.responseComments.add(responseRow);
+                } else {
+                    InstructorAttributes instructor = results.roster.getInstructorForEmail(giverEmail);
+                    boolean isInstructorGiver = giverEmail.equals(instructor.email);
+                    boolean isAllowedToSubmitSessionsInBothSection = instructor.isAllowedForPrivilege(response.giverSection,
+                            response.feedbackSessionName,
+                            Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS)
+                            && instructor.isAllowedForPrivilege(response.recipientSection, response.feedbackSessionName,
```

Comment:
These two lines should not be on the same level: `Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS` is a parameter of `instructor.isAllowedForPrivilege`. Perhaps a better way would be to shift `instructor.isAllowedForPrivilege` down, and indent the parameters an additional 8 spaces.